### PR TITLE
Update _2021-05-14-opensearch-beta-announcement.markdown

### PR DESCRIPTION
**Pull request by ahopp**
- Updated language in announcement and thanks
- Removed superfluous content and killed artifacts header
- Add feature callout to announcement
- Flagged community meeting as needing update

### Description
Continue to improve the announcement
 
### Issues Resolved
None specifically

By submitting this pull request, I confirm that my contribution is made under the terms of the BSD-3-Clause License.

### CODE REVIEW

File: _posts/_2021-05-14-opensearch-beta-announcement.markdown [hunk 1]
```diff
@@ -81,4 +69,5 @@ If you're excited to jump in, we've marked a few issues that would be really hel
 
 **Questions? Feedback?**
 
-Let us know in the [forums](https://discuss.opendistrocommunity.dev/). We are also having biweekly [Community Meetings](https://www.meetup.com/Open-Distro-for-Elasticsearch-Meetup-Group/). The community meetings will include progress updates at every session as well as leaving time for Q&A. The next community meeting is on May 6, 2021 at 7:30 AM Pacific Time (UTC -8).
+Let us know in the [forums](https://discuss.opendistrocommunity.dev/). We are also having biweekly [Community Meetings](https://www.meetup.com/Open-Distro-for-Elasticsearch-Meetup-Group/). The community meetings will include progress updates at every session as well as leaving time for Q&A. The next community meeting is on May 18, 2021 at
```
Comment (elfisher) [inline _posts/_2021-05-14-opensearch-beta-announcement.markdown:72]: Let's also link customers to github issues on the opersearch repo as an option to give feedback

File: _posts/_2021-05-14-opensearch-beta-announcement.markdown [hunk 2]
```diff
@@ -8,68 +8,56 @@ date:   2021-05-14 01:01:01 -0700
...
+- SQL Query Syntax
+- Anomaly Detection
+- Alerting
+- Piped Processing Language
+
+...and more ([OpenSearch](https://github.com/opensearch-project/OpenSearch) and [OpenSearch Dashboards](https://github.com/opensearch-project/OpenSearch-Dashboards)). 
+
+We would like to get your feedback from this beta release - please let us know if there are any features you would like or bugs you identify. You can submit pull requests, write documentation, open issues (either on [OpenSearch](https://github.com/opensearch-project/OpenSearch/issues) or [OpenSearch Dashboards](https://github.com/opensearch-project/OpenSearch-Dashboards/issues)), or simply read the [news](https://opensearch.org/blog/). If you would like to install or run this project please see the Developer Guide ([OpenSearch](https://github.com/opensearch-project/OpenSearch/blob/main/DEVELOPER_GUIDE.md) and [OpenSearch Dashboards](https://github.com/opensearch-project/OpenSearch-Dashboards/blob/main/DEVELOPER_GUIDE.md)) to get started.
+
+#### Plugins Included in Beta are:
```
Comment (elfisher) [inline _posts/_2021-05-14-opensearch-beta-announcement.markdown:23]: let's just make the name of the plugin the link to the repo too.

**Review by elfisher — CHANGES_REQUESTED** [review]
some minor feedback.

File: _posts/_2021-05-14-opensearch-beta-announcement.markdown (hunk 1 above)
Comment (ahopp) [reply _posts/_2021-05-14-opensearch-beta-announcement.markdown:72]: Great idea - I'll add.

File: _posts/_2021-05-14-opensearch-beta-announcement.markdown (hunk 2 above)
Comment (ahopp) [reply _posts/_2021-05-14-opensearch-beta-announcement.markdown:23]: Makes sense - I'll update this next.

Comment (ahopp) [reply _posts/_2021-05-14-opensearch-beta-announcement.markdown:23]: Done!

File: _posts/_2021-05-14-opensearch-beta-announcement.markdown (hunk 1 above)
Comment (ahopp) [reply _posts/_2021-05-14-opensearch-beta-announcement.markdown]: Done!

**Comment by ahopp** [comment]
Made a few updates, including;
- added the users who have posted in the Forking category (forum) and made contributions (GitHub)
- wordsmithed the "thank you" slightly
- resolved missing plugin URLs

**Comment by ahopp** [comment]
> @ahopp can you make this PR to staging?

Done, thanks!

**Pull request merged**